noxfile.py

```diff
@@ -98,25 +98,10 @@ def test(session):
     session.run("python", "-m", "eland.tests.setup_tests")
     session.run("pytest", "--doctest-modules", *(session.posargs or ("eland/",)))
 
-    session.run("python", "-m", "pip", "uninstall", "--yes", "scikit-learn", "xgboost")
+    session.run("python", "-m", "pip", "uninstall", "--yes", "lightgbm")
```

Comment:
What makes `lightgbm` a special case here?

Reply:
Whoops, this should be uninstalling all the libraries to ensure they're optional. I'll fix that.